packages/cyberstorm/src/components/Toast/ToastContainer.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+import { Toast } from "./Toast";
+import styles from "./Toast.module.css";
+import * as RadixToast from "@radix-ui/react-toast";
+
+export function ToastContainer(props: {
+  toasts: {
+    id: string;
```

Comment:
The comments made in `Toast.tsx` regarding the props and their naming obviously apply in here and in `ToastComplex`. In fact defining the interface once and importing it to where it's actually needed would be a valid reason to export it in the first place (currently it's exported but never used anywhere). Note that we can export it internally within `@thunderstore/cyberstorm`, without exporting it outside of the package.

Reply:
Changed all imports to use one type definition